infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -34,6 +34,20 @@ You could set the environment variables for variables to stored terraform backen
 `cp backend.tfvars.template backend.tfvars`.
 2. Edit the backend.tfvars to configure the necessary TF backend state.
 
+## Docker setup
+
+To be able to build the Tables and Housetables service images, you will need to ensure Docker is running on your local machine. This is typically done by opening Docker Desktop. You can check for this by running a command like `docker info` in the terminal.
```

Comment:
```suggestion
To be able to build the Tables and Housetables service images, you will need to ensure Docker is installed on your local machine. 
```

Reply:
It needs to be both installed and currently running; the image build doesn't work if I don't have the docker desktop open